projects/extension/src/background/ConnectionManager.ts

```
@@ -133,6 +133,8 @@ export class ConnectionManager extends (EventEmitter as { new(): StateEmitter })
           type: 'basic'
         });
       });
+    } finally {
```

Why do you need a try/finally here?

---

I need a `try` block in order to silently discard problematic ports, otherwise the whole extension would stop if a port contains a wrong name.
`try { ... }` alone isn't valid JS, you need either `catch` or `finally`, so I put `finally`.

---

See my review comment about using `catch` and adding an error message.